pkg/resource/gcs.go

```diff
@@ -0,0 +1,224 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"sync"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+
+	"github.com/leaktk/scanner/pkg/logger"
+	"github.com/leaktk/scanner/pkg/response"
+)
+
+var getClientMu sync.Mutex
+var storageClient *storage.Client
```

Comment:
Absolutely inconsequential, but you can group `var` declarations:
```suggestion
var (
    getClientMu sync.Mutex
    storageClient *storage.Client
)
```

Reply:
I always forget about that in go. Thanks for the note!